jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -169,6 +169,14 @@ public final void setMaxMultiplex(int maxMultiplex)
         if (maxMultiplex < 1)
             throw new IllegalArgumentException("Max multiplex must be >= 1");
         this.maxMultiplex = maxMultiplex;
```

Comment:
Should we rename the private field to make it clear it is the default:
```suggestion
        this.defaultMaxMultiplex = maxMultiplex;
```

Eitherway, this class now needs more javadoc describing how the maxMultiplex works

Reply:
We don't have the word "default" for a lot of other cases, e.g. `ServerConnector.idleTimeout` and `EndPoint.idleTimeout`, or the flow control windows got HTTP/2 `ConnectionFactory`s, so I would not add "default" here.
It is just the `maxMultiplex` of the `Pool`.

Reply:
But in other places we don't have the same fieldname on a subclass in the same file, but with a different meaning.

Reply:
Also at the very least we need javadoc